DuckDuckGo/NetworkProtectionStatusViewModel.swift

```diff
@@ -106,6 +109,13 @@ final class NetworkProtectionStatusViewModel: ObservableObject {
             .map(Self.statusImageID(connected:))
             .assign(to: \.statusImageID, onWeaklyHeld: self)
             .store(in: &cancellables)
+        isConnectedPublisher
```

Comment:
Possibly nitpicking, but this is not really the best place to put this. If the user taps connect and then navigates off the screen before it completes, the connection won’t get registered. I know that might be a statistically irrelevant number of cases, but it more makes the point that this is possibly not the best place to put this.

You could alternatively put this into the `NetworkProtectionPacketTunnelProvider` as that’s really the source of truth for when it’s activated.